### PR TITLE
Design Rework #5 - New Icons

### PR DESCRIPTION
**Pull request by LuisMossburger**
**Description**
See also #100 and #118 for progress.
This merges the use of two new icon styles derived from four icon packs by Freepik (Flaticon).
* Use colorful, detailed icons for "inside" (list views etc.)
* Use black outline icons for "outside" (text editor, navigation, header)
* Add an imprint fragment to give proper attribution & a navgation item for that in every fragment

Next up: Toasts & Help, Home Fragment, "design sugar".

**Affects**
App design, therefore everyone can take a look. CC @xSabrina @silvia-ivanova-github-rep @claudia3 

**Notes for Reviewer**
Please review & comment/approve. Thanks!

### CODE REVIEW

File: BibBuddy/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -272,12 +279,28 @@
     <string name="filter_note">Notizen</string>
 
 
+    <!-- Imprint Fragment -->
+    <string name="headerImprint">Impressum</string>
```
Comment (SarahKurek) [inline BibBuddy/app/src/main/res/values/strings.xml:283]:
```suggestion
    <string name="header_imprint">Impressum</string>
```

**Review by SarahKurek — COMMENTED** [review]
Please rename the commented name. 
The black sort and library icon is not identifiable as such to me, but we can clarify that in the meeting.

**Comment by LuisMossburger** [comment]
> Please rename the commented name.
> The black sort and library icon is not identifiable as such to me, but we can clarify that in the meeting.

I have written this down for the TOPs of tomorrows meeting!

**Review by SarahKurek — APPROVED** [review]
Thank you for the changes. Now it fits.

**Pull request merged**